src/FormItem.tsx

```diff
@@ -18,25 +19,34 @@ export const FormItem = <TFieldValues extends FieldValues = FieldValues>({
   control,
   name,
   help,
+  valuePropName,
   ...props
 }: FormItemProps<TFieldValues>) => {
-  const { field, fieldState } = useController({ name, control });
-
-  const handleNormalize: AntdFormItemProps['normalize'] = (value) => {
-    field.onChange(value);
-    return value;
-  };
+  const { fieldState } = useController({ name, control });
 
   return (
-    <AntdForm.Item
-      {...props}
+    <Controller
```

Comment:
Could you try [useController](https://react-hook-form.com/docs/usecontroller) instead of `Controller` element to make the code less complex?

Reply:
Sure, I guess it was redundant after all.